### PR TITLE
Feature/animation

### PR DESCRIPTION
**Pull request by RadoslawStepniewski**
Added animation to images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -207,7 +207,9 @@ <h1 class="lubisz">Lubisz piwo?</h1>
             <div class="row">
                 <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
                     <div class="osoba">
+
```
Comment (burunia) [inline index.html]: remove white space

File: css/style.css [hunk 2]
```diff
@@ -149,13 +148,26 @@ body {
...
 .fotka {
     margin: 0 auto;
-    width: 100px;
+    width: 50%;
+    border: #ffffff 5px solid;
+    transition-duration:0.6s;
+    transition-timing-function: ease;
+
+}
+.fotka:hover{
```
Comment (patrykomiotek) [inline css/style.css]: Why are you writing CSS selectors in Polish?

File: css/style.css [hunk 3]
```diff
@@ -149,13 +148,26 @@ body {
...
+    background: #000000;
+    border: #007227 5px solid;
+    width: 60%;
+
 }
 
+
```
Comment (patrykomiotek) [inline css/style.css]: Same as above

File: index.html [hunk 4]
```diff
@@ -321,9 +323,9 @@ <h1 class="lubisz">Lubisz piwo?</h1>
 <div class="footer">&copy; All rights reserved&emsp;-&emsp;<span class="big">Lubeer</span></div>
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
 <script>
-    $(document).ready(function(){
+    $(document).ready(function () {
```
Comment (patrykomiotek) [inline index.html:323]: Have you tried without jQuery first? ;)

**Review by patrykomiotek — CHANGES_REQUESTED** [review]
Keep classes and CSS selectors in English

**Review by burunia — APPROVED** [review]

**Pull request merged**